com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/lib/domain/ApplicationAttributesUpdateBehvavior.java

```diff
@@ -0,0 +1,27 @@
+package com.sap.cloud.lm.sl.cf.client.lib.domain;
+
+public class ApplicationAttributesUpdateBehvavior {
```

Comment:
You are correct. But I am with a mix feelings here. First, I really like the solution with the nested class. On the other hand, I cry when I see the CloudApplicationExtended because we are overflow it. ;( I will move it